Sources/WhoopDIKit/Container/Container.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+
 public final class Container {
     private let localDependencyGraph: ThreadSafeDependencyGraph
```

Comment:
Should this be removed now?

Reply:
we can remove it if we are certain we want task local to be the final way (then we can even remove the options in total or make it empty for now)